lib/rubocop/extension/generator/generator.rb

```diff
@@ -122,10 +122,7 @@ def self.defaults!
                 exit!
               end
 
-              github_user = `git config github.user`.chop
-              github_user = 'your_id' if github_user.empty?
-
-              generator = RuboCop::Cop::Generator.new(cop_name, github_user)
+              generator = RuboCop::Cop::Generator.new(cop_name)
```

Comment:
Can you add the following to rubocop-extension-generator.gemspec that it can dependent on RuboCop API changes?

```diff
+ spec.add_runtime_dependency 'rubocop', '>= 1.22.0'
  spec.add_runtime_dependency 'bundler'
  spec.add_runtime_dependency 'activesupport'
```

Reply:
Add at fbd70bc